sam_view.c

```diff
@@ -560,211 +560,428 @@ static const char *copied_tags[] = { "RG", "BC", "QT", NULL };
 
 static void bam2fq_usage(FILE *to)
 {
-    fprintf(to, "\nUsage:   samtools bam2fq [-nOt] [-s <outSE.fq>] <in.bam>\n\n");
-    fprintf(to, "Options: -n        don't append /1 and /2 to the read name\n");
-    fprintf(to, "         -O        output quality in the OQ tag if present\n");
-    fprintf(to, "         -s FILE   write singleton reads to FILE [assume single-end]\n");
-    fprintf(to, "         -t        copy RG, BC and QT tags to the FASTQ header line\n");
-    fprintf(to, "\n");
+    fprintf(to,
+"Usage:   samtools bam2fq [-nOt] [-s <outSE.fq>] <in.bam>\n\n"
+"Options: -n        don't append /1 and /2 to the read name\n"
+"         -O        output quality in the OQ tag if present\n"
+"         -v INT    default quality score if not given in file [1]\n"
+"         -s FILE   write singleton reads to FILE [assume single-end]\n"
+"         -t        copy RG, BC and QT tags to the FASTQ header line\n"
+"         -1 FILE   write paired reads flagged READ1 to FILE\n"
+"         -2 FILE   write paired reads flagged READ2 to FILE\n"
+"         -0 FILE   write paired reads flagged both or neither READ1 and READ2 to FILE\n"
+"         -f INT    only include reads with all bits set in INT set in FLAG [0]\n"
+"         -F INT    only include reads with none of the bits set in INT\n"
+"                   set in FLAG [0]\n"
+"         -T TYPE   write the output in FASTQ or FASTA [FASTQ]\n");
```

Comment:
As you're reformatting all this usage source code and adding lots of options, please take the opportunity to format the usage as

    Options:
      -n     don't blah blah
      -O     blah blah

i.e., hyphens under `Options` indented by **two** spaces: this maximises width utilisation and lines up with `sam_global_opt_help()`.  Indentation of `don't`/`blah`/etc stays as appropriate for whatever `INT`/`FILE`/etc option arguments the subcommand has.